core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -125,13 +126,14 @@ var TopicsAndSkillsDashboardPage = function() {
     });
   };
 
-  this.createTopicWithTitle = function(title) {
+  this.createTopicWithTitle = function(title, abbreviatedName) {
```

Comment:
Should this be renamed to this.createTopicWithTitleAndAbbreviatedName (or possibly just this.createTopic)?

Reply:
Changed this to createTopic. Thanks!